### ISSUE TITLE
[SUGGESTION] Add mod of item to tooltip

### ISSUE THREAD

**Issue by joendter** [Gliese-832-c/ad-astra-per-nucleon#24]
**Is your suggestion related to a problem? Please describe.**
When I hover over an item i only see it's name but not it's mod. But if I do so in a crafting recipe it shows the mod

**Describe the solution you'd like**
Also add the mod for normal inventory tooltip

**Additional context**
![2021-01-27_11 05 27](https://user-images.githubusercontent.com/71767955/105975759-a0d78200-608f-11eb-97c9-e01bf2c1f261.png)
![2021-01-27_11 05 34](https://user-images.githubusercontent.com/71767955/105975767-a2a14580-608f-11eb-87a2-1fe1ca169390.png)

**Comment by Gliese-832-c** [comment]
Yeah, it's a JEI feature. I should probably add a mod that shows it in the normal inventory. For now, you can exit your inventory and press F3+H to show the ID, which will show you the mod.
